### PR TITLE
Toast Overview: change demo description

### PR DESCRIPTION
**Pull request by vladaskorohodova**

### CODE REVIEW

File: JSDemos/Demos/Toast/Overview/description.md [hunk 1]
```diff
@@ -1 +1,25 @@
-The Toast is&nbsp;a&nbsp;UI component that provides pop-up notifications. It&nbsp;can be&nbsp;displayed in&nbsp;one of&nbsp;the following modes: _&laquo;info&raquo;_, _&laquo;warning&raquo;_, _&laquo;error&raquo;_, _&laquo;success&raquo;_ and _&laquo;custom&raquo;_. In&nbsp;this demo, the _&laquo;success&raquo;_ and _&laquo;error&raquo;_ modes are used depending on&nbsp;the context.
\ No newline at end of file
+The Toast is a UI component that provides pop-up notifications.
```
Comment (dmitry-eliseev-devexpress) [inline JSDemos/Demos/Toast/Overview/description.md]: Just be careful with the word "provides": https://tw.devexpress.devx/language-guidelines/overused-words/provide/

```suggestion
The Toast is a UI component that displays pop-up notifications. 
```

File: JSDemos/Demos/Toast/Overview/description.md [hunk 2]
```diff
@@ -1 +1,25 @@
-The Toast is&nbsp;a&nbsp;UI component that provides pop-up notifications. It&nbsp;can be&nbsp;displayed in&nbsp;one of&nbsp;the following modes: _&laquo;info&raquo;_, _&laquo;warning&raquo;_, _&laquo;error&raquo;_, _&laquo;success&raquo;_ and _&laquo;custom&raquo;_. In&nbsp;this demo, the _&laquo;success&raquo;_ and _&laquo;error&raquo;_ modes are used depending on&nbsp;the context.
\ No newline at end of file
+The Toast is a UI component that provides pop-up notifications. 
+
+When you need to display a notification, call the [notify(message, type, displayTime)](/Documentation/ApiReference/Common/Utils/ui/#notifymessage_type_displayTime) method with values for the [message](/Documentation/ApiReference/UI_Components/dxToast/Configuration/#message), [type](/Documentation/ApiReference/UI_Components/dxToast/Configuration/#type) and [displayTime](/Documentation/ApiReference/UI_Components/dxToast/Configuration/#displayTime) properties passed as the arguments.
+
+You can specify one of the four predefined types of notifications, depending on the mood of the message:
+
+- *'info'*   
+Blue toast with a message bubble icon.
```
Comment (dmitry-eliseev-devexpress) [inline JSDemos/Demos/Toast/Overview/description.md]: Just a blue toast. Mind the articles, please.

```suggestion
A blue toast with a message bubble icon.
```

File: JSDemos/Demos/Toast/Overview/description.md [hunk 3]
```diff
@@ -1 +1,25 @@
...
+
+- *'warning'*   
+Yellow toast with an exclamation mark icon.
```
Comment (dmitry-eliseev-devexpress) [inline JSDemos/Demos/Toast/Overview/description.md]:
```suggestion
A yellow toast with an exclamation mark icon.
```

File: JSDemos/Demos/Toast/Overview/description.md [hunk 4]
```diff
@@ -1 +1,25 @@
...
+
+- *'error'*   
+Red toast with an **X** icon.
```
Comment (dmitry-eliseev-devexpress) [inline JSDemos/Demos/Toast/Overview/description.md]:
```suggestion
A red toast with an **X** icon.
```

File: JSDemos/Demos/Toast/Overview/description.md [hunk 5]
```diff
@@ -1 +1,25 @@
...
+
+- *'success'*   
+Green toast with a check mark icon.
```
Comment (dmitry-eliseev-devexpress) [inline JSDemos/Demos/Toast/Overview/description.md]:
```suggestion
A green toast with a check mark icon.
```

File: JSDemos/Demos/Toast/Overview/description.md [hunk 6]
```diff
@@ -1 +1,25 @@
...
+
+In this demo, toggle the checkboxes to see the *'success'* and *'error'* types of notifications.
```
Comment (dmitry-eliseev-devexpress) [inline JSDemos/Demos/Toast/Overview/description.md]:
```suggestion
In this demo, toggle check boxes to see the *'success'* and *'error'* notification types.
```

**Review by dmitry-eliseev-devexpress — DISMISSED** [review]

File: JSDemos/Demos/Toast/Overview/description.md [hunk 7]
```diff
@@ -1 +1,25 @@
-The Toast is&nbsp;a&nbsp;UI component that provides pop-up notifications. It&nbsp;can be&nbsp;displayed in&nbsp;one of&nbsp;the following modes: _&laquo;info&raquo;_, _&laquo;warning&raquo;_, _&laquo;error&raquo;_, _&laquo;success&raquo;_ and _&laquo;custom&raquo;_. In&nbsp;this demo, the _&laquo;success&raquo;_ and _&laquo;error&raquo;_ modes are used depending on&nbsp;the context.
\ No newline at end of file
+The Toast is a UI component that displays pop-up notifications. 
+
+When you need to display a notification, call the [notify(message, type, displayTime)](/Documentation/ApiReference/Common/Utils/ui/#notifymessage_type_displayTime) method with values for the [message](/Documentation/ApiReference/UI_Components/dxToast/Configuration/#message), [type](/Documentation/ApiReference/UI_Components/dxToast/Configuration/#type) and [displayTime](/Documentation/ApiReference/UI_Components/dxToast/Configuration/#displayTime) properties passed as the arguments.
```
Comment (arminal) [inline JSDemos/Demos/Toast/Overview/description.md]:
```suggestion
When you need to display a notification, call the [notify(message, type, displayTime)](/Documentation/ApiReference/Common/Utils/ui/#notifymessage_type_displayTime) method with values for the [message](/Documentation/ApiReference/UI_Components/dxToast/Configuration/#message), [type](/Documentation/ApiReference/UI_Components/dxToast/Configuration/#type), and [displayTime](/Documentation/ApiReference/UI_Components/dxToast/Configuration/#displayTime) properties passed as arguments.
```

File: JSDemos/Demos/Toast/Overview/description.md [hunk 8]
```diff
@@ -1 +1,25 @@
...
+
+If you need to specify other Toast properties besides [type](/Documentation/ApiReference/UI_Components/dxToast/Configuration/#type) and [displayTime](/Documentation/ApiReference/UI_Components/dxToast/Configuration/#displayTime), call the [notify(options, type, displayTime)](/Documentation/ApiReference/Common/Utils/ui/#notifyoptions_type_displayTime) method and pass an object as the argument. In this object, you can set any Toast property, such as [shading](/Documentation/ApiReference/UI_Components/dxToast/Configuration/#shading), [position](/Documentation/ApiReference/UI_Components/dxToast/Configuration/#position), [width](/Documentation/ApiReference/UI_Components/dxToast/Configuration/#width), [height](/Documentation/ApiReference/UI_Components/dxToast/Configuration/#height) and [others](/Documentation/ApiReference/UI_Components/dxToast/Configuration/).
```
Comment (arminal) [inline JSDemos/Demos/Toast/Overview/description.md]:
```suggestion
If you need to specify other Toast properties in addition to [type](/Documentation/ApiReference/UI_Components/dxToast/Configuration/#type) and [displayTime](/Documentation/ApiReference/UI_Components/dxToast/Configuration/#displayTime), call the [notify(options, type, displayTime)](/Documentation/ApiReference/Common/Utils/ui/#notifyoptions_type_displayTime) method and pass an object as the argument. In this object, you can set any Toast property, such as [shading](/Documentation/ApiReference/UI_Components/dxToast/Configuration/#shading), [position](/Documentation/ApiReference/UI_Components/dxToast/Configuration/#position), [width](/Documentation/ApiReference/UI_Components/dxToast/Configuration/#width), [height](/Documentation/ApiReference/UI_Components/dxToast/Configuration/#height), and [others](/Documentation/ApiReference/UI_Components/dxToast/Configuration/).
```

**Review by arminal — COMMENTED** [review]
#GrammarReview

Made edits to comma usage and phrasing.

**Review by DokaRus — APPROVED** [review]
LGTM

**Pull request merged**